project-base/storefront/utils/cart/useAddToCart.ts

```diff
@@ -59,20 +59,6 @@ export const useAddToCart = (
         dispatchBroadcastChannel('refetchCart');
 
         const addedCartItem = addToCartResult.addProductResult.cartItem;
-        const notOnStockQuantity = addToCartResult.addProductResult.notOnStockQuantity;
-
-        if (notOnStockQuantity > 0) {
-            showErrorMessage(
-                t(
-                    'You have the maximum available amount in your cart, you cannot add more (total {{ quantity }} {{ unitName }})',
```

Comment:
please run `pnpm run translate` in the storefront container to update the translation files